test/e2e/inspect_test.go

```diff
@@ -289,4 +289,99 @@ var _ = Describe("Podman inspect", func() {
 		Expect(baseJSON[0].HostConfig.SecurityOpt).To(Equal([]string{"label=type:spc_t,label=level:s0", "seccomp=unconfined"}))
 	})
 
+	It("podman inspect pod", func() {
+		podName := "testpod"
+		create := podmanTest.PodmanNoCache([]string{"pod", "create", "--name", podName})
+		create.WaitWithDefaultTimeout()
+		Expect(create.ExitCode()).To(Equal(0))
+
+		inspect := podmanTest.Podman([]string{"inspect", podName})
+		inspect.WaitWithDefaultTimeout()
+		Expect(inspect.ExitCode()).To(Equal(0))
+		Expect(inspect.IsJSONOutputValid()).To(BeTrue())
+		podData := inspect.InspectPodArrToJSON()
+		Expect(podData[0].Name).To(Equal(podName))
+	})
+
+	It("podman inspect pod with type", func() {
+		podName := "testpod"
+		create := podmanTest.PodmanNoCache([]string{"pod", "create", "--name", podName})
+		create.WaitWithDefaultTimeout()
+		Expect(create.ExitCode()).To(Equal(0))
+
+		inspect := podmanTest.Podman([]string{"inspect", "--type", "pod", podName})
+		inspect.WaitWithDefaultTimeout()
+		Expect(inspect.ExitCode()).To(Equal(0))
+		Expect(inspect.IsJSONOutputValid()).To(BeTrue())
+		podData := inspect.InspectPodArrToJSON()
+		Expect(podData[0].Name).To(Equal(podName))
+	})
+
+	It("podman inspect latest pod", func() {
+		podName := "testpod"
+		create := podmanTest.PodmanNoCache([]string{"pod", "create", "--name", podName})
+		create.WaitWithDefaultTimeout()
+		Expect(create.ExitCode()).To(Equal(0))
+
+		inspect := podmanTest.Podman([]string{"inspect", "--type", "pod", "--latest"})
```

Comment:
Will not work on Remote.

Add 
if IsRemote() {
inspect := podmanTest.Podman([]string{"inspect", "--type", "pod", podName})
} else {
inspect := podmanTest.Podman([]string{"inspect", "--type", "pod", "--latest"})
}